bitnami/minio/README.md

```diff
@@ -516,6 +455,10 @@ Find more information about how to deal with common errors related to Bitnami's
 
 ## Upgrading
 
+### To 12.1.0
+
+All gateway features have been removed from Minio since upstream completely dropped this feature.
+
```

Comment:
```suggestion
### To 12.0.0

This version updates MinIO&reg; to major version 2023.  All gateway features have been removed from Minio since upstream completely dropped this feature. The related options have been removed in version 12.1.0.

```